README.md

```diff
@@ -5,23 +5,46 @@
 This is the backend for the Simple app to help track hypertensive patients across a population.
 
 ## Development Setup
-First, you need to install ruby: https://www.ruby-lang.org/en/documentation/installation
-It is recommended to use rbenv to manage ruby versions: https://github.com/rbenv/rbenv
+
+To set up the Simple server for local development, clone the git repository and
+run the setup script included.
+
+```
+$ git clone git@github.com:simpledotorg/simple-server.git
+$ cd simple-server
+$ bin/setup
```

Comment:
Just double-checking, is the `bin/setup` script intended for use?

Reply:
It is something we've used before. It came as an open-source contribution from outside.